systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -284,6 +286,51 @@ void testHostAliases() {
         assertThat(output, containsString(etcHostsData));
     }
 
+    @Test
+    void testConfigureDeploymentStrategy() {
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
```

Comment:
Kafka is created in `beforeAll`, do we need this "replace" ?

Reply:
Yes, my bad... thanks for this spot!